Remotion/Core/Core/Utilities/DateOnlyConverter.cs

```diff
@@ -0,0 +1,73 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Remotion.Utilities;
+
+public class DateOnlyConverter : TypeConverter
+{
+  private readonly Type _dateTimeType = typeof(DateTime);
+  private readonly Type _nullableDateTimeType = typeof(DateTime?);
+  private readonly Type _nullableDateOnlyType = typeof(DateOnly?);
+  private readonly Type _dateOnlyType = typeof(DateOnly);
+
+  public override bool CanConvertFrom (ITypeDescriptorContext? context, Type sourceType)
+  {
+    ArgumentUtility.CheckNotNull("sourceType", sourceType);
+
+    return IsDateTimeType(sourceType);
+  }
+
+  public override bool CanConvertTo (ITypeDescriptorContext? context, Type? destinationType)
+  {
+    return IsDateTimeOrDateOnlyType(destinationType);
```

Comment:
isn't this a mismatch with `CovnertTo` since there we only handle DateTimeType, not DateOnlyType.

Reply:
Yea, but I'm not sure whether we want to change it here or in the convertTo method

Reply:
If we don't need the DateOnly-test, we should drop it completely and the associated logic. removes complexity and if it's not needed, it's not needed. There's also "no" TypeConverter that does a same-type conversion as far as I'm aware.